watch/watch.c

```diff
@@ -32,25 +33,24 @@ static char *progname;
 int main(int argc, char *argv[]);
 void usage(void);
 
-struct Flag {
-	int Beep_on_error, No_title;
-}; static struct Flag flag;
-
 int main(int argc, char *argv[]) {
 	progname = argv[0];
 	int option = 0;
-	struct timespec interval = {0};
-	int c = 0, ec = 0, term_x = 0, term_y = 0;
+	int fBeep_on_error = 0,
```

Comment:
I like the decision to remove the structure.

Reply:
I did not remove the struct, just changed the line where it was declared to be after ``pid_t exec_pid;``.
https://github.com/Projeto-Pindorama/heirloom-ng/blob/watch-implementation-changes/watch/watch.c#L45

Reply:
> I did not remove the struct, just changed the line where it was declared to be after `pid_t exec_pid;`. https://github.com/Projeto-Pindorama/heirloom-ng/blob/watch-implementation-changes/watch/watch.c#L45

I mean the one containing `Beep_on_error` and `No_title`